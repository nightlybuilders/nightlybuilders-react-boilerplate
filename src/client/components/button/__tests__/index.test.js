import React from 'react'
import { Button } from '../'

describe('Components/Button', () => {
  test('renders without throwing an error', () => {
    expect(<Button>Click Me</Button>).toMatchSnapshot()
  })
})
