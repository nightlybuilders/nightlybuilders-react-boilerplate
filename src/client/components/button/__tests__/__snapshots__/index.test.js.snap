// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Button renders without throwing an error 1`] = `
<Button
  className=""
  onClick={[Function]}
  styles={Object {}}
>
  Click Me
</Button>
`;
